python/desc/imsim/imSim.py

```diff
@@ -763,7 +764,10 @@ def read_config(config_file=None):
     if config_file is None:
         config_file = os.path.join(lsstUtils.getPackageDir('imsim'),
                                    'data', 'default_imsim_configs')
-    cp.read(config_file)
+
+    if not cp.read(config_file):
+        raise FileNotFoundError("Config file {} not found".format(config_file))
+
```

Comment:
GitHub isn't letting me comment on the line since it hasn't changed, but below here (on line 776) you are still calling read_config (right after the function definition).  But you are now doing this on line 57 in bin/imSim.py.

Will this wipe out any parameters that were changed programmatically since then?

Reply:
The `read_config` here picks up the defaults so that I can pop into ipython and not have to call `read_config` explicitly if I want to do something simple, like read the observing parameters from an instance catalog or construct an obs_md object from it.   The `read_config` call at https://github.com/LSSTDESC/imSim/blob/u/jchiang/read_config_fix/bin.src/imsim.py#L57 ensures that any user provided config is read in and will be used when the code is executed.